### PR TITLE
Jz add img processing example

### PR DESCRIPTION
**Pull request by jingzhi93**
When you added a new program / programs , put a cross on the  OS that you have test it on  
(Example: [ X ] Windows  )

[ X ] Windows  
[  ] Linux Ubuntu  
[  ] Mac  
[  ] Others  (State here -> xxx )  

Assign a reviewer to review your code.

### CODE REVIEW

File: dl4j-cv-labs/src/main/java/global/skymind/solution/image_processing/Quantization.java [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+        Mat src = imread(imgpath, IMREAD_GRAYSCALE);
+        NativeImageLoader loader = new NativeImageLoader();
+        INDArray imgArray = loader.asMatrix(src);
+        int width = src.arrayWidth();
+        int height = src.arrayHeight();
+
+        //reduce intensity
+        INDArray resultImgArray = reduceDepth(imgArray, width, height, bit);
+        Display.display(src, "original");
+        JPanelDisplay display = new JPanelDisplay(resultImgArray, "" + bit + " image");
```
Comment (kenghooi-teoh) [inline dl4j-cv-labs/src/main/java/global/skymind/solution/image_processing/Quantization.java:67]: @jingzhi93 do you intend to print `bit + "bit image" `?

File: dl4j-cv-labs/src/main/java/global/skymind/solution/image_processing/MedianFilter.java [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+        Mat saltpepperNoise = new Mat();
+        threshold(randomMat, saltpepperNoise, 250, 255, THRESH_BINARY);
+
+        //add noise to the original image
+        Mat imgNoised = img.clone();
+        add(img, saltpepperNoise, imgNoised);
+
+        //perform median filtering on the image with noise
+        Mat filteredImg = new Mat();
+        medianBlur(imgNoised, filteredImg, 3);
```
Comment (kenghooi-teoh) [inline dl4j-cv-labs/src/main/java/global/skymind/solution/image_processing/MedianFilter.java:65]: we have this in ImageBlur.java, can we combine them to become one exercise?

**Comment by kenghooi-teoh** [comment]
@jingzhi93  reminder to fetch updates from master

File: dl4j-cv-labs/src/main/java/global/skymind/solution/image_processing/Quantization.java (hunk 1 above)
Comment (jingzhi93) [reply dl4j-cv-labs/src/main/java/global/skymind/solution/image_processing/Quantization.java]: nope just display it on the frame

File: dl4j-cv-labs/src/main/java/global/skymind/solution/image_processing/MedianFilter.java (hunk 2 above)
Comment (jingzhi93) [reply dl4j-cv-labs/src/main/java/global/skymind/solution/image_processing/MedianFilter.java]: i tried this before but the effect is not that obvious

**Comment by jingzhi93** [comment]
@kenghooi-teoh done

**Review by kenghooi-teoh — APPROVED** [review]

**Pull request merged**